cmd/check_container.go

```diff
@@ -229,3 +229,16 @@ func init() {
 
 	checkCmd.AddCommand(checkContainerCmd)
 }
+
+func buildConnectURL(projectID string) string {
+	connectURL := fmt.Sprintf("https://connect.redhat.com/projects/%s/overview", projectID)
+	pyxisHost := viper.GetString("pyxis_host")
+
+	if len(pyxisHost) > 0 {
+		s := strings.Split(pyxisHost, ".")
```

Comment:
I don't think this will work in all cases. pyxisHost should never be 0 length, I think. It has a default set. So this will always happen. If you didn't set PYXIS_HOST, the s[1] below would be 'redhat'. We might instead need to have a separate 'PFLT_PYXIS_ENV' or something.